src/Nest/Mapping/Types/Specialized/Attachment/AttachmentProperty.cs

```diff
@@ -19,6 +20,7 @@ public interface IAttachmentProperty : IDocValuesProperty
 		ITextProperty LanguageField { get; set; }
 	}
 
+	[Obsolete("Removed in 6.0.")]
```

Comment:
I think we can help users here by pointing them to using the `ingest-attachment` plugin in the comment.

Reply:
:+1: